Update: property page to Javascript property page (#25847)
mdn/content: files/en-us/glossary/object_reference/index.md

A link to an **{{glossary("object")}}**. Object references can be used exactly like the linked objects.

> Object reference concepts become clear when assigning the same object to more than one _[property](Glossary/Property/JavaScript)_. Rather than holding a copy of the object, each assigned property holds object references that link to the same object so that when the object changes, all properties referring to the object reflect the change.

Reviewer: I like the original.

```suggestion
The concept of object references become clear when assigning the same object to more than one _[property](Glossary/Property/JavaScript)_. Rather than holding a copy of the object, each assigned property holds object references that link to the same object so that when the object changes, all properties referring to the object reflect the change.
```

Reviewer 2: Reverted.

> The concept of object references becomes clear when assigning the same object to more than one _ {{glossary("property/javascript")}}_. Rather than holding a copy of the object, each assigned property holds object references that link to the same object, so that when the object changes all properties referring to the object reflect the change.

Reviewer 2: ```suggestion
The concept of object references becomes clear when assigning the same object to more than one _{{glossary("property/javascript")}}_. Rather than holding a copy of the object, each assigned property holds object references that link to the same object, so that when the object changes, all properties referring to the object reflect the change.
```

> The concept of object references becomes clear when assigning the same object to more than one _{{glossary("property/javascript")}}_. Rather than holding a copy of the object, each assigned property holds object references that link to the same object, so that when the object changes, all properties referring to the object reflect the change.

Reviewer: Does `glossary()` remove the `property/` slug?

Reviewer 2: Checking...

Reviewer 2: ```suggestion
The concept of object references becomes clear when assigning the same object to more than one _[property](Glossary/Property/JavaScript)_. Rather than holding a copy of the object, each assigned property holds object references that link to the same object, so that when the object changes, all properties referring to the object reflect the change.
```

> The concept of object references becomes clear when assigning the same object to more than one **{{glossary("property/javascript")}}**. Rather than holding a copy of the object, each assigned property holds object references that link to the same object, so that when the object changes all properties referring to the object reflect the change.

Reviewer 2: Removing bold for emphasis.

```suggestion
The concept of object references becomes clear when assigning the same object to more than one _ {{glossary("property/javascript")}}_. Rather than holding a copy of the object, each assigned property holds object references that link to the same object, so that when the object changes all properties referring to the object reflect the change.
```
